src/fragments/orders.ts

```diff
@@ -154,6 +154,7 @@ export const fragmentOrderLine = gql`
 export const fragmentOrderLineWithMetadata = gql`
   fragment OrderLineWithMetadata on OrderLine {
     ...OrderLine
+    ...Metadata
```

Comment:
How about loading line-related metadata when i click "view metadata"? - not for now, but it's always a bit of optimization

Reply:
Right, it makes sense I'll put it into separate PR 👍🏻